lib/web_ui/lib/src/engine/navigation/history.dart

```diff
@@ -183,15 +183,15 @@ class MultiEntriesBrowserHistory extends BrowserHistory {
   }
 
   @override
-  void onPopState(covariant DomPopStateEvent event) {
+  void onPopState(covariant DomPopStateEvent state) {
```

Comment:
Can we keep it the name `event` or make it `Object? state`?

I have a feeling you were trying to change `DomPopStateEvent event ` ==> `Object? state` but forgot to finish the change, or reverted it partially.

Reply:
Hmmm... you're right, I kind of did a half-measure here. I think I can clean this up a bit.

Reply:
So what I ended up doing here is to basically unwrap the JS event in `JSUrlStrategy` before it actually gets to the darty `UrlStrategy` object, and so the pop state listener just takes the state itself. I think this is a bit better. For stuff on the framework side that uses actual JS interop to do the pop listener thing, they can just unwrap it there as well. The path through JSUrlStrategy remains unchanged, so I think this is non-breaking for the framework (but I'll find out as soon as I start testing against the framework side).